src/index.ts

```diff
@@ -5,22 +5,33 @@ import path from 'node:path'
 import { findRelease } from './release'
 import { checkerName, version } from './shared'
 
+const WORKING_DIR = path.join(process.env.HOME ?? "/tmp", 'editorconfig-checker')
```

Comment:
Is there any reason to do not use /tmp by default?

Couldn't we use a temporary working directory? Something equivalent to  `mktemp -d` (shell) or `os.TempDir` (Go) ?

Reply:
If we would like to use `/tmp`, we should use `os.tmpdir()`.

https://nodejs.org/docs/latest/api/os.html#ostmpdir

Reply:
I‘ve used that as a fallback because the environment variable could be undefined. I doubt that it ever will be, but yeah, we should guard against it.